### ISSUE TITLE
Searching for a Logo

### ISSUE THREAD

**Issue by Wikunia** [Wikunia/Javis.jl#95]
With v0.1.0 being released in a the next days we should have an issue for ideas for a potential logo.

**Comment by cormullion** [comment]
Gotta start somewhere:

![javis](https://user-images.githubusercontent.com/52289/94336382-c28a0500-ffda-11ea-845d-96ff0e284636.png)

**Comment by Wikunia** [comment]
That's a great start @cormullion it's somehow messing  abit with my eyes 😂 The red leaks into the center which disappears when I zoom in. There seems to be a grey line between red and green. 

I might add some ideas here the next week. Let the game for searching begin! 😊

**Comment by cormullion** [comment]
Ha, just a sketch at this stage (although at Float64 precision). 😂

**Comment by cormullion** [comment]
Ah, I missed the first anniversary of this Issue... :) Anyway, I had a few hours this morning and jotted down a few more ideas to stimulate your thoughts. I got stuck for a while thinking about the "eye" for "visualization", and then managed to escape to film/video/production and beyond.
 
![javis-assembly](https://user-images.githubusercontent.com/52289/131222166-fe4bdef4-f032-43e3-88ef-2c1af42fe03c.png)

Hopefully this will provoke someone into coming up with some better ideas! 😅🤔

**Comment by cormullion** [comment]
No worries! The trouble with most of these is that they're too complicated to make good logos. Making things as simple as possible is very difficult for me... 😄

<img width="480" alt="Screenshot 2021-10-18 at 11 00 10" src="https://user-images.githubusercontent.com/52289/137710550-4a97462d-ee3b-46e7-8733-b3a52b14bae6.png">

**Comment by Wikunia** [comment]
Yeah for something like a favicon they are too detailed. I actually meant to have only a single `J` (replacing the three frames). This one works as well though. Maybe (to make it even more complicated) I would change something like color/position such that it would not be a still image. Yes I'm bad at simplifying as well.
However if we really only have one J and change the background to a single color then the logo might be relatively simple.

**Comment by cormullion** [comment]
Sort of like this?

<img width="362" alt="Screenshot 2021-10-18 at 12 16 38" src="https://user-images.githubusercontent.com/52289/137720699-e54c1712-39c8-464d-a82b-045b0c3c7c04.png">

**Comment by Wikunia** [comment]
Exactly. I like it :smile:
